### DISCUSSION TITLE
是否可以关闭系统托盘图标

### DISCUSSION THREAD

**Discussion by chenzhesi** [electerm/electerm#2465] [Q&A] [answered]
win10系统，之前使用的1.20.10没有系统托盘图标，升级到1.21.34后，打开程序会默认出现系统托盘图标，在设置里面也没有找到关闭的方法，请问有方法关闭吗？

**Comment by zxdong262** [comment]
没有这个设置，windows系统栏设置可以隐藏吧，请问为什么要关闭呢

**Reply by chenzhesi** [reply]
因为这个托盘图标相对主界面没有添加任何其他功能，为了任务栏简洁想关掉（平时会把所有的托盘图标都显示出来不隐藏）

**Comment by zxdong262** [comment, marked as answer]
已经在最新版去掉了
